### PR TITLE
E2E: Use smaller test images

### PR DESCRIPTION
**Pull request by WunderBart**
#### Changes proposed in this Pull Request

We're currently using unnecessarily large images for testing purposes:

- [image0.jpg](https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/image-uploads/image0.jpg) ~200kB
- [image1.jpg](https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/image-uploads/image1.jpg) ~75kB
- [image2.jpg](https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/image-uploads/image2.jpg) ~150kB

I suggest using [smaller ones](https://github.com/Automattic/wp-calypso/tree/update/e2e-test-files/test/e2e/image-uploads) (~220 bytes each) that will not only make tests faster but also save some local/hosting space after many test runs 😉 

Screenshot 2021-12-02 at 15 34 15

#### Testing instructions

All tests should pass.

### CODE REVIEW

**Comment by WunderBart** [comment]
@worldomonation do you think we should be deleting those temporary media files in the test teardown hook?

**Comment by worldomonation** [comment]
> @worldomonation do you think we should be deleting those temporary media files in the test teardown hook?

Hm, I swear I had implemented something for that already but my memory must have failed me.

It could be a worthwhile addition to the spec itself to remove any test files that were generated, though I would assign it a priority level of 'normal' for the task.

By the way, I think the images you show in the original description were generated by the Selenium suite, as for the Playwright suite I use a timestamp instead.

File: test/e2e/specs/constants.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-	path.join( __dirname, '..', 'image-uploads', 'image0.jpg' )
+	path.join( __dirname, '..', 'image-uploads', 'test-image-01.png' )
 );
 export const TEST_AUDIO_PATH = path.normalize(
-	path.join( __dirname, '..', 'image-uploads', 'bees.mp3' )
+	path.join( __dirname, '..', 'image-uploads', 'test-audio-01.mp3' )
 );
-export const UNSUPPORTED_FILE_PATH = path.normalize(
-	path.join( __dirname, '..', 'image-uploads', 'unsupported_extension.mkv' )
+export const TEST_UNSUPPORTED_FILE_PATH = path.normalize(
```
Comment (worldomonation) [inline test/e2e/specs/constants.js]: Thanks for aligning this with other names.

**Review by worldomonation — APPROVED** [review]

File: test/e2e/specs/constants.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import path from 'path';
 
 export const TEST_IMAGE_PATH = path.normalize(
-	path.join( __dirname, '..', 'image-uploads', 'image0.jpg' )
+	path.join( __dirname, '..', 'image-uploads', 'test-image-01.png' )
```
Comment (scinos) [inline test/e2e/specs/constants.js]: For the record, `path.join()` calls `path.normalize()` internally, so this is redundant.

**Comment by WunderBart** [comment]
> I think the images you show in the original description were generated by the Selenium suite, as for the Playwright suite I use a timestamp instead.

Ah, makes sense! I haven't cleared the repo since the Selenium sessions! 😅

File: test/e2e/specs/constants.js (hunk 2 above)
Comment (WunderBart) [reply test/e2e/specs/constants.js]: Removed in https://github.com/Automattic/wp-calypso/pull/58752/commits/5ee9718561dc0f3b273cbfb283b086b2e0206e97

**Pull request merged**